src/docs/development/ui/assets-and-images.md

```diff
@@ -38,7 +38,10 @@ flutter:
 ```
 
 {{site.alert.note}}
-  Only files located directly in the directory are included unless there are files with the same name inside a subdirectory (see Asset Variants). To add files located in subdirectories, create an entry per directory.
+  Only files located directly in the directory are included unless
+  there are files with the same name inside a subdirectory 
+  (see Asset Variants). To add files located in subdirectories, create 
```

Comment:
Should I add a link to Asset Variants topic here?

Reply:
Yes please! And thanks for breaking up that massive line!

Reply:
I have added the link